recipes/systemd_service.rb

```diff
@@ -8,19 +8,20 @@ class ::Chef::Recipe
 node.default["chef_client"]["bin"] = client_bin
 create_directories
 
-dist_dir, conf_dir = value_for_platform_family(
-  ["fedora"] => ["fedora", "sysconfig"],
-  ["rhel"] => ["redhat", "sysconfig"]
+dist_dir, conf_dir, env_file = value_for_platform_family(
+  ["arch"] => ["arch", "chef", "arch.env"],
```

Comment:
With the deprecation of conf.d, the recommended way in Arch is to refer to the program's config files [1]. That's why I moved it to /etc/chef.

[1] https://wiki.archlinux.org/index.php/DeveloperWiki:Systemd#Units